static/tps/2019_1/tp3/index.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: page
+title: "TP3 – Grafos y la red de delicuencia"
+permalink: '/tps/2019_1/tp3'
+math: true
+---
+
+Trabajo Práctico 3
+==================
+
+{:.no_toc}
+
+El Trabajo Práctico número 3 es de elaboración grupal, y tiene fecha de entrega
+para el **viernes 28 de junio**.
+
+Contenido
+---------
+{:.no_toc}
+
+* Contenido
+{:toc}
+
+## Introducción
+
+Luego de una ardua investigación por parte del FBI sobre una red de delicuentes, lograron encontrar 
+una serie de mensajes enviados que los pueden llegar a comprometer. Lograron obtener los datos de 
+más de 420.000 mensajes enviados por distintos medios: algunos son mensajes de texto, otros llamadas 
+telefónicas, otros mensajitos por diversas redes sociales, mails, etc... Todos estos conectan a 260.000 
+sospechosos a nivel mundial. El problema que ahora enfrentan es que no saben bien cómo trabajar con 
+estos datos, por lo que le pidieron a alumnes de la FIUBA si pueden ayudar con esta tarea.
+
+## Análisis de la situación
+
+Los agentes del FBI se contactaron pidieron que se implementen las siguientes funcionalidades (o 
+_features_):
+1. Tienen a algunos de los delicuentes menores bien vigilados, y saben que se van a contactar con los 
+demás. Eventualmente, si siguen el rastro pueden acabar encontrando y atrapando a algún _pez gordo_ de 
+la lista de más buscados del mundo. Por esto, quieren poder obtener la mínima cantidad de seguimientos 
+de personas que debe realizar un agente para dar con dicho _pez gordo_. Por ejemplo, ese tal 
+_Toni, el Gordo_ es un delicuente muy buscado en una ciudad de Estados Unidos. 
+1. Con tantos datos obtenidos, no tienen forma de definir cuáles son los delicuentes más importantes, 
+para saber contra quienes conviene apuntar en esta operación. 
+1. En función de lo anterior, les intersa poder tener una lista de cuales pueden ser esos delicuentes.
+El problema es que son tantos, que saben que un algoritmo exacto puede tomar demasiado tiempo en calcular
+esto, y temen que los delicuentes se enteren que el FBI ha obtenido todos estos datos, y escapen. 
+Por lo tanto, aceptan una buena aproximación. 
+1. Otra cosa que desean es, dado una lista de delicuentes a los que están siguiendo en estos momentos, 
+determinar cuál es el mínimo esfuerzo para encontrar a alguno de los delicuentes más importantes.
+1. Algo más que les permitiría entender como está conformada esta red de delicuentes sería poder determinar 
+comunidades dentro de esta red.
+1. Además de esto, cuentan con agentes infiltrados. Les interesa saber, si alguno de estos agentes comienza 
+una noticia/rumor, hasta dónde puede llegar!
+1. En conjunto con lo anterior, les interesa poder saber si existe algún camino de un determinado largo que 
+empiece en un delicuente en particular (que en realidad será un agente encubierto), y que termine en dicho 
```

Comment:
Perdón, no puedo hacer más sugerencias por _delicuente_. Plz apply a global search-and-replace.

Reply:
Definitivamente tengo que revisar qué le sucede a algunas letras de ese teclado...